### ISSUE TITLE
XML Comments should not contain empty lines

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#414]
A comment such as
```c#
/// <summary>
/// Some comment here.
///
/// </summary>
```

should trigger a violation because of the empty line.
The proposed fix should be to use a `/// <para/>` instead.
